### ISSUE TITLE
Alt+Shift+comma emit Alt + comma, it should emit Alt + [less than sign]

### ISSUE THREAD

**Issue by etern** [microsoft/terminal#1819]
# Environment

```none
Windows build number: Microsoft Windows [版本 10.0.18362.175]
Windows Terminal version (if applicable): Preview installed from Windows Store

Any other software?

- Ubuntu wsl installed from Windows Store
- emacs in wsl
```
# Steps to reproduce

Add wsl to Windows Terminal profile, can't type `Alt-<` `Alt->` `Alt-:` `Alt-?`，these are usual commands in emacs.

And these keys works all right in Ubuntu wsl
# Expected behavior
`Alt-Shift-,` emit `Alt-<`
`Alt-Shift-.` emit `Alt->`

# Actual behavior
`Alt-Shift-,` emit `Alt-,`
`Alt-Shift-.` emit `Alt-.`

**Comment by DHowett-MSFT** [comment]
Yeah, this is definitely an issue. We'll need something slightly better than `towlower` for this one.

**Comment by DHowett-MSFT** [comment]
Fixed in #2836.
